ci-operator/config/openshift/service-catalog/master.json

```diff
@@ -21,7 +21,7 @@
     "tag": "golang-1.10"
   },
 
-  "canonical_go_repository": "github.com/kubernetes-incubator/service-catalog",
+  "canonical_go_repository": "github.com/openshift/service-catalog",
```

Comment:
No...

Reply:
canonical_go_repository has to match the Go import paths inside the repo.

Reply:
updated both jobs, ptal